### PR TITLE
doctest faq.md

### PR DESCRIPTION
**Pull request by KristofferC**
Some sections didnt make sense to put into a doctest (like those that had generated code as output). I updated them to what matched from my computer however.

### CODE REVIEW

File: doc/src/manual/faq.md [hunk 1]
```diff
@@ -386,52 +392,54 @@ to aggressively optimize simple little functions like `f(k) = 5k-1`. The machine
...
 ```
 
-The actual body of the function is a single `lea` instruction, which computes the integer multiply
+The actual body of the function is a single `leaq` instruction, which computes the integer multiply
 and add at once. This is even more beneficial when `f` gets inlined into another function:
 
 ```julia
-julia> function g(k,n)
+julia> function g(k, n)
          for i = 1:n
```
Comment (tkelman) [inline doc/src/manual/faq.md]: should use 4 space indent here

File: doc/src/manual/faq.md [hunk 2]
```diff
@@ -444,17 +452,17 @@ julia> function g(k)
 g (generic function with 2 methods)
```
Comment (tkelman) [inline doc/src/manual/faq.md]: and here

**Comment by KristofferC** [comment]
updated

**Comment by KristofferC** [comment]
any more comments here?

**Pull request merged**